SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -450,9 +450,8 @@ void USpatialWorkerConnection::Flush()
 	{
 		ProcessOutgoingMessages();
 	}
-	else
+	else if (ensure(ThreadWaitCondition.IsSet()))
```

Comment:
Shouldn't this be one of those ensureAlways? But I am not sure how these work with compilers.

Reply:
ensureAlways just means that a callstack is generated *everytime* this condition is false. Whereas ensure will only do it the first time in a session.